### ISSUE TITLE
Set up #370 feature/upgradeFrontend2 on test server

### ISSUE THREAD

**Issue by Don-Isdale** [plantinformatics/pretzel#381]
Make #370   feature/upgradeFrontend2 available for testing on dirac

---
- [x] depends on #380 

---

- [x] use a duplicate of the existing pretzel database
- [x] includes feature/useBrAPI and feature/upgradeFrontend

**Comment by Don-Isdale** [comment]
#### Completed

The current version of feature/upgradeFrontend2 is running on the test server in URL sub-directory /pretzelUpdate

During testing it was noted that it is not possible to have 2 Pretzel web applications running in the same browser - the session is closed and the user is bounced back to the login page.  This is not a concern for testing the frontend upgrade.
